packages/rest/src/lib/global/fetch.d.ts

```diff
@@ -0,0 +1,5 @@
+import type * as undici from 'undici';
```

Comment:
I tried it and it worked fine for me (cause I was worried it would be a problem). I suppose we can mark it as a normal dep and it can either get shook away or simply not actually imported at runtime.